packages/keystone/src/fields/types/relationship/views/cards/index.tsx

```diff
@@ -75,23 +75,22 @@ export function Cards({
 }: {
   foreignList: ListMeta;
   localList: ListMeta;
-  field: ReturnType<typeof controller> & { display: { mode: 'cards' } };
-  id: string;
-  forceValidation: boolean | undefined;
+  id: string | null;
   value: { kind: 'cards-view' };
```

Comment:
@mitchellhamilton curiosity: Is there value in providing a more explicit type here, or is this purely for type narrowing purposes?

Reply:
It's to narrow the type